.github/workflows/lint-pr.yml

```diff
@@ -1,22 +0,0 @@
-name: "Lint PR"
-
-on:
-  pull_request_target:
-    types:
-      - opened
-      - edited
-      - synchronize
-
-permissions:
-  contents: read
-
-jobs:
-  main:
-    permissions:
-      pull-requests: read  # for amannn/action-semantic-pull-request to analyze PRs
-      statuses: write  # for amannn/action-semantic-pull-request to mark status of analyzed PR
```

Comment:
This is just checking if the PR name is following the semantic format, I dont think we need this

Reply:
(Wasn't linting in the first place)